src/assert.js

```diff
@@ -295,20 +295,9 @@ class Assert {
 		let actual,
 			result = false;
 
-		const currentTest = ( this instanceof Assert && this.test ) || config.current;
+		[ expected, message ] = validateExpectedExceptionArgs( expected, message, "throws/raises" );
```

Comment:
It's a bit unrelated, but feel free to omit `raises` from the error message here. We support it as a legacy alias and that doesn't have to go anywhere, but I think keeping it here adds more confusion than it removes. People that had to use it, presumably know that it is an alias and can work it out :)